tests/test_application.coffee

```diff
@@ -6,16 +6,24 @@ via the new configuration api introduced in space:base.
 
 class @CustomerApp extends Space.Application
 
+  @publish -> 'CustomerApp'
+
   RequiredModules: ['Space.eventSourcing']
 
   Dependencies: {
     commandBus: 'Space.messaging.CommandBus'
     eventBus: 'Space.messaging.EventBus'
-    Mongo: 'Mongo'
+    mongo: 'Mongo'
   }
 
   Configuration: {
-    useSnapshotting: true
+    testMode: true
```

Comment:
Not yet so will remove. I'll document the Configuration API [here](https://meteor-space.readme.io/v1.0/docs/configuration) which can be our master reference, including a specification of what has been implemented